locust_plugins/jmeter_listener.py

```diff
@@ -136,11 +140,20 @@ def add_result(self, success, _request_type, name, response_time, response_lengt
             connect,
         ]
         self.csv_results.append(self.field_delimiter.join(row))
-        if len(self.csv_results) >= self.flush_size:
+        if len(self.csv_results) >= self.flush_size and self.client_type != "WorkerRunner":
```

Comment:
Use typeof instead of string comparison

Reply:
do you mean isinstance? - If so, I think this needs an explicit import to be able to use it.

Reply:
I meant `type(thing) is WorkerRunner` (not typeof, that is Java :)

isinstance or type() work equally well in this case. Yes, they require an import, but it is still nicer than string compare...